### ISSUE TITLE
Documentation and installation

### ISSUE THREAD

**Issue by zeevox** [ColtonZ/learning_hub#1]
Would it be possible to add information to the README on how to run this project.
Also what licence is this code licensed under?

**Comment by harens** [comment]
Good point @ZeevoX. If you could add build instructions, I'd be happy to try and add a Travis Config file to help build and deploy the project.
